packages/tasks/src/hardware.ts

```diff
@@ -168,6 +176,14 @@ export const SKUS = {
 				tflops: 9.22,
 				memory: [6],
 			},
+			"GTX 1080 Ti": {
+				tflops: 11.3,
+				memory: [12],
+			},
+			"GTX 1070 Ti": {
+				tflops: 1.62,
```

Comment:
```suggestion
				tflops: 8.2,
```

Using float32 performance from https://www.techpowerup.com/gpu-specs/geforce-gtx-1070-ti.c3010